client/src/resources/cert-and-project-map.ts

```diff
@@ -40,7 +40,7 @@ const legacyInfosecQaInfosecBase = infoSecBase;
 
```

Comment:
```suggestion
type CertMap = {
  id: string;
  title: string;
  certSlug: string;
  flag: string;
  projects: Readonly<{
    id: string;
    title: string;
    link: string;
    certSlug: string;
  }[]> | null;
};
```

Reply:
The issue here is we lose the strong typing. Considering nothing here is dynamically created, it would be better to find a solution that does not lose us this information.